src/test/java/uk/gov/companieshouse/overseasentitiesapi/converter/OverseasEntitySubmissionDaoConverterTest.java

```diff
@@ -31,7 +31,22 @@ class OverseasEntitySubmissionDaoConverterTest {
     private DocumentTransformerFactory transformerFactory;
 
     @Test
-    void testDocumentConversion() {
+    void testDocumentConversionFor1_0() {
```

Comment:
Based on a comment from @markpit on my earlier review, suggest changing these method names slightly to improve readability. E.g. `testDocumentConversionFor_1_0_Model`

Reply:
Names changed